### PR TITLE
Dust Managed Credentials

### PR DESCRIPTION
**Pull request by spolu**
- roll out for managed data sources in product
- roll out for API when using a system Key (for now only applicable for data sources)

nit: removes line-clamp plugin as it's now included in tailwind latest versions.

Deploy:
- Update `env.local` for `front`
- Deploy `front`

### CODE REVIEW

File: front/types/provider.ts [hunk 1]
```diff
@@ -2,3 +2,15 @@ export type ProviderType = {
   providerId: string;
   config: string;
 };
+
+export type CredentialsType = {
```
Comment (fontanierh) [inline front/types/provider.ts:6]: Can you delete from [here](https://github.com/dust-tt/dust/blob/d3e25aa2f72c4ae9bb76fd7fef4d10c9c23b3bc2/front/lib/providers.ts#L171-L181) 🙏

**Review by fontanierh — APPROVED** [review]
🚀

File: front/types/provider.ts (hunk 1 above)
Comment (spolu) [reply front/types/provider.ts:6]: uh already done I moved it from there :)

Comment (fontanierh) [reply front/types/provider.ts:6]: Oh my bad missed that 🤦

**Pull request merged**